src/app-logic.js

```diff
@@ -11,7 +11,7 @@ import { PROPOSAL_STATUS_ACTIVE_STRING } from './constants'
 export default function useAppLogic() {
   const { account } = useWallet()
 
-  const { isLoading, stakeToken, totalStaked } = useAppState()
+  const { isLoading, stakeToken, totalStaked, errorFetchingApp, errorFetchingOrg } = useAppState()
```

Comment:
What do you think if we only expose an `errors` variable from the AppStatePRovider which combines the logic of both errors?